CHANGELOG.md

```diff
@@ -55,6 +55,17 @@ With this feature, you can specify the local time zone of the service or system
 
 We have also extended the time zone options so you can select any of the standard [ISO 8601 time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
 
+# 7.0.0-beta3 (2020-05-08)
+
+### Features / Enhancements
+* **Docker**: Upgrade to Alpine 3.11. [#24056](https://github.com/grafana/grafana/pull/24056), [@aknuds1](https://github.com/aknuds1)
+* **Forms**: Remove Forms namespace [BREAKING]. [#24378](https://github.com/grafana/grafana/pull/24378), [@tskarhed](https://github.com/tskarhed)
```

Comment:
@tskarhed can you provide some details here about the  possible consequences of this change?